pages/post/[postname].js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import moment from 'moment';
+import Link from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons';
+import MarkdownIt from 'markdown-it';
+import Layout from '../../components/Layout';
+import Primary from '../../components/Primary';
+import CloudinaryImage from '../../components/CloudinaryImage';
+import Parser from '../../components/Parser';
+import theme from '../../theme';
+
+export default function BlogPost({ attributes, markdownBody }) {
+  const md = new MarkdownIt();
+
+  if (!attributes) return <></>;
+  return (
+    <StyledBlogPost>
+      <Layout>
+        <Primary>
+          <article>
+            <h1>{attributes.title}</h1>
+            <div className="blog__post__date">Published on {moment(attributes.date).format('MMMM D, YYYY')}</div>
+            <div className="blog__post__image">
+              {attributes.hero && (
+                <CloudinaryImage
+                  url={attributes.hero}
+                  alt={attributes.title}
+                  maxWidth={theme.newsItemImageWidthPixels}
+                />
+              )}
+            </div>
+            <div className="blog__post__authors">
+              By:&thinsp;
+              {attributes.authors.length > 1
+                ? attributes.authors.map(author => <span key={author}>{md.renderInline(author)}</span>)
+                : attributes.authors}
+            </div>
+            <div>
+              <Parser>{markdownBody}</Parser>
```

Comment:
I'd love to explore some different typographic treatments for the body of these blog posts. At first blush, line height feels a little cramped and line length feels a little long, making the posts tricky to read.

Reply:
I'm going to take a stab at improving the typography here using the [style guide](https://docs.google.com/document/d/1n0QzA7jBB9lzAmrGDQvcIJYHZ5O1i_Zu0G1oHQW0cNc/edit#).